flow_screen_components/clipboardButton/README.md

```diff
@@ -0,0 +1,43 @@
+# clipboardButton
+
+Lightning Web Component for Flow Screens: **clipboardButton**
+
+Lightning Flow Builder Name: **Copy to Clipboard Button**
+
+** This clipboardButton component is designed to be used on a FLow Screen where the value passed to the component**
+** is displayed and when the button is clicked, the value will be inserted into the system clipboard.**
+
+Additional components packaged with this LWC:
+
+                    Lightning Web Components:   clipboardButtonCPE
+
+**Documentation:** https://unofficialsf.com/copy-to-clipboard-button/
+
+**Created by:** Eric Smith  
+**Date:** March 23, 2021
+
+LinkedIn: https://www.linkedin.com/in/ericrsmith2  
+Salesforce: https://trailblazer.me/id/ericsmith  
+Blog: https://ericsplayground.wordpress.com/blog/  
+Twitter: https://twitter.com/esmith35
+
+---
+
+**You must install these components FIRST in order to install and use the Datatable component**  
```

Comment:
This guidance is a little confusing. Should we not just add Datatable as a third dependency you need to install? I do that in one extension...

Reply:
My fault.  That's leftover text from a copy/paste.  I'll remove it and upload a quick PR.